backend/dal/Helpers/Extensions/PropertyExtensions.cs

```diff
@@ -36,7 +37,7 @@ public static IQueryable<Entity.Views.Property> GenerateQuery(this PimsContext c
             // Users may only view sensitive properties if they have the `sensitive-view` claim and belong to the owning agency.
             var query = context.Properties
                 .AsNoTracking()
-                .Where(p => p.ClassificationId != 4); // Disposed properties are not visible.
+                .Where(p => p.ClassificationId != (int)ClassificationTypes.Classifications.DISPOSED && p.ClassificationId != (int)ClassificationTypes.Classifications.SUBDIVIDED && p.ClassificationId != (int)ClassificationTypes.Classifications.DEMOLISHED); // Disposed and subdivided/demolished properties are not visible.
```

Comment:
You don't have to change, but `!=` are actually poor performing queries in SQL.  It is always better to be explicit `=` or `in`.  In code however it is easier to read and write, especially if the condition is for a single value.

Reply:
Did we have requirements around admin users being able to view subdivided parcels? actually, I think that was just in the generated reports.